packages/commerce-sdk-react/package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@salesforce/commerce-sdk-react",
-  "version": "4.0.0-extensibility-preview.1",
+  "version": "4.0.0-extensibility-preview.5",
```

Comment:
the current version in package 4.0.0-extensibility-preview.5, so the package lock is corrected when I run `npm i`